src/sql/azdata.d.ts

```diff
@@ -2319,6 +2319,11 @@ declare module 'azdata' {
 		 */
 		displayName: string;
 
+		/**
+		 * Display language id of provider
+		 */
+		languageId: string;
```

Comment:
This should probably go to azdata proposed

Reply:
yeah, all changes to `azdata.d.ts` should start in proposed and then move to the stable API once we fully are committed to maintain the change as part of the long-term extensibility API.